PokeAlarm/Events/MonEvent.py

```diff
@@ -337,3 +333,20 @@ def generate_dts(self, locale, timezone, units):
                 and self.weight <= 2.41 else '')
         })
         return dts
+
+    def _disappear_time_infos(self, time, spawn_verified):
```

Comment:
This seems like something that would do better in the PokeAlarm.Utils file

Reply:
Sure :)

Reply:
Sorry, I meant PokeAlarm.MonUtils since this will only be used for monsters :smile: